net/shorewall6-lite/Makefile

```diff
@@ -28,7 +28,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/shorewall6-lite
     SECTION:=net
     CATEGORY:=Network
-    DEPENDS:=+ip +iptables6 +shorewall-core
+    DEPENDS:=+ip +iptables +shorewall-core
```

Comment:
Found the problem.
Messed up my build system. After switching off IPv6, switching it on does not show the ip6* options. Doing a 'menu defconfig' solves it.

After installing I have all the right packages:
ip                 ip6tables-restore  ipcalc.sh          iptables-restore
ip6tables          ip6tables-save     iptables           iptables-save